swiss-ng-api.yaml

```diff
@@ -5975,6 +5978,11 @@ components:
         endToEndIdentification:
           type: string
           maxLength: 35
+#        instructionIdentification: # N.A. for all, but can be used by ASPSP if needed
+#          type: string
+#          maxLength: 35
+#        debtorName: # N.A. for all, but can be used by ASPSP if needed
+#          $ref: "#/components/schemas/debtorName"
```

Comment:
debtorName is mandatory for all. See line 5921 (new file) or 5920 (old file). These lines should NOT be in comments.

Here you are editing the properties of the model paymentInitiation_json (line 5890). Make sure you are editing the properties of the correct model.

Reply:
You're right, we'll be corrected.